src/Module.php

```diff
@@ -49,9 +49,9 @@ public function init(ModuleManagerInterface $manager)
     /**
      * loadModulesPost callback
      *
-     * @param  $event
+     * @param EventInterface $event
      */
-    public function onLoadModulesPost($event)
+    public function onLoadModulesPost(EventInterface $event)
```

Comment:
Please do not add native php typehints in this PR as this introduces a BC break. 
Focus on PHP 8.0 compatibility only.
Thanks!